docs/GettingStarted.md

```diff
@@ -497,47 +472,39 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | Key | Description | Default |
 | --- | ----------- | ------- |
 | `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `false` |
-| `service_name` | Service name used for `action_mailer` instrumentation | `'action_mailer'` |
 | `email_data` | Whether or not to append additional email payload metadata to `action_mailer.deliver` spans. Fields include `['subject', 'to', 'from', 'bcc', 'cc', 'date', 'perform_deliveries']`. | `false` |
 
-### Active Model Serializers
+### Action Pack
 
-The Active Model Serializers integration traces the `serialize` event for version 0.9+ and the `render` event for version 0.10+.
+Most of the time, Action Pack is set up as part of Rails, but it can be activated separately:
 
 ```ruby
-require 'active_model_serializers'
+require 'actionpack'
 require 'ddtrace'
 
 Datadog.configure do |c|
-  c.use :active_model_serializers, options
+  c.use :action_pack
 end
-
-my_object = MyModel.new(name: 'my object')
-ActiveModelSerializers::SerializableResource.new(test_obj).serializable_hash
 ```
 
-| Key | Description | Default |
-| --- | ----------- | ------- |
-| `service_name` | Service name used for `active_model_serializers` instrumentation. | `'active_model_serializers'` |
-
-### Action Pack
+### Action View
 
-Most of the time, Action Pack is set up as part of Rails, but it can be activated separately:
+Most of the time, Active Support is set up as part of Rails, but it can be activated separately:
```

Comment:
This was already like this, but should it be "Most of the time, **Action View** is set up..." rather than _Active Support_ like it is now?

Reply:
Likely was an artifact of rebasing. Fixed.